tools/compatgen/internal/tensorflow.go

```diff
@@ -45,7 +51,7 @@ func FetchTensorFlowCompatibilityMatrix() ([]config.TFCompatibility, error) {
 	}
 
 	// sanity check
-	if len(compats) < 21 {
+	if len(compats) < 12 {
 		return nil, fmt.Errorf("Tensorflow compatibility matrix only had %d rows, has the html changed?", len(compats))
 	}
```

Comment:
Since this is now parameterized, this should no longer return an error. We should either: 

1. Remove / replace it with a warning later on if no solutions are found, or
2. Print an error instead of returning an error